packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import $ from '@js/core/renderer';
+import Button from '@js/ui/button';
+
+import TextArea from '../m_text_area';
+import Widget from '../widget';
+
+const CHAT_MESSAGE_BOX_CLASS = 'dx-chat-message-box';
+const CHAT_MESSAGE_BOX_TEXTAREA_CLASS = 'dx-chat-message-box-text-area';
+const CHAT_MESSAGE_BOX_BUTTON_CLASS = 'dx-chat-message-box-button';
+
+class MessageBox extends Widget<any> {
+  // TODO chech ts
```

Comment:
Why you don't use types from textArea.d.ts?

Reply:
[90dd3acd3c6dd8b06c21608a237d774ee8b6aa43]